sim/pokemon.ts

```diff
@@ -1780,7 +1779,7 @@ export class Pokemon {
 	}
 
 	hasAbility(ability: string | string[]) {
-		if (this.ignoringAbility()) return false;
+		if (this.ignoringAbility(ability === 'klutz')) return false;
```

Comment:
Strictly speaking callers of `hasAbility` aren't required to pass in an id (see use of `toID` below).